packages/eslint-config-kodefox/package.json

```
@@ -11,25 +11,25 @@
   "author": "devteam@kodefox.com",
   "license": "MIT",
   "peerDependencies": {
-    "eslint": ">=7.0.0",
-    "prettier": ">=2.0.0"
+    "eslint": ">=7.30.0",
+    "prettier": ">=2.3.2"
   },
   "optionalDependencies": {
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-react-native": "^3.10.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.9.0",
-    "@typescript-eslint/parser": "^4.9.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
+    "@typescript-eslint/parser": "^4.28.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-prettier": "^3.1.4"
   },
   "devDependencies": {
-    "eslint": "^7.14.0",
-    "prettier": "^2.2.1"
+    "eslint": ">=7.30.0",
+    "prettier": ">=2.3.2"
```

Let's keep the caret here
```suggestion
    "eslint": "^7.30.0",
    "prettier": "^2.3.2"
```